src/Recovery/Common/src/HttpClient/CurlClient.php

```diff
@@ -81,6 +81,7 @@ private function call($url, $method = self::METHOD_GET, array $header = [], $dat
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($curl, CURLOPT_FOLLOWLOCATION, true);
         curl_setopt($curl, CURLOPT_HEADER, true);
+        curl_setopt($curl, CURLOPT_USERAGENT, "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:74.0) Gecko/20100101 Firefox/74.0");
```

Comment:
is there a reason, it has to be firefox?

Reply:
No, theres no specific reason. It could be any inconspicuous User-Agent.

On the other side - is there a reason, it should not be firefox? I think not.

Reply:
does something like `Shopware Installer` also work?

Reply:
I think `Shopware Installer` doesn't look like a typical Bot, so it is unlikely, that it will be a part of some blocking patterns. Feel free to change it to your suggestion!